src/components/networked-hand-controls.js

```diff
@@ -153,38 +106,269 @@ AFRAME.registerComponent('networked-hand-controls', {
+    // possible future enhancement: add more detailed thumbstick info
+    originalFn(buttonName, evtName);
+  },
+
+  injectRemoteControllerModel() {
+    // this is for a networked controller entity, where we want to show a non-hand controller model
+    // we add the relevant controller component, and then hack it just a bit to:
+    // A) convince it the model show without relation to whether one is actually locally connected, and 
+    // B) to not track _local_ pose data (so, no prevent/remove tracked-controller component)
+
+    // we pause the element so we can prevent pose tracking from being added.
+    this.el.pause();
+
+    // we add the actual component, to get the model generated and button meshes and event handling
+    this.el.setAttribute(this.data.controllerComponent, {
+      hand: this.data.hand, 
+      model: true,
+      // could also optionally support buttonColor, buttonTouchColor, buttonHighlightColor 
+    });
+
+    // we don't want the remote model to listen to local button/trigger/joystick events, though 
+    this.el.components[this.data.controllerComponent].removeEventListeners();
+    this.el.components[this.data.controllerComponent].removeControllersUpdateListener();
+
+    // some older controller components aren't as well written, and so we have to allow pose tracking to be added and then remove it
+    if (!this.el.components[this.data.controllerComponent].checkIfControllerPresent ||
+        !this.el.components[this.data.controllerComponent].loadModel
+        ) {
+      console.warn("fallback path: will add and then disable pose tracking", this.data.controllerComponent, this.el, this.el.components);
+
+      // however, we do still want _one_ event listener, so we add the button meshes
+      this.el.addEventListener('model-loaded', this.el.components[this.data.controllerComponent].onModelLoaded);
+
+      // here we have no loadModel() to use directly, so we fully inject (bypassing controller detection)
+      this.el.components[this.data.controllerComponent].injectTrackedControls({profiles:JSON.parse(this.data.webxrControllerProfiles)});
+      this.disableLoadedPoseTracking();
+    } 
+    else {
+      // ideal, default path, for controller components (like oculus-touch-controls), that have separate loadModel() function
+
+      // this prevents injectTrackedControllers from running, which prevents the 
+      // tracked-controls-webxr component from being added, which is responsible for pose tracking
+      // (which we are disabling, since we want this controller to follow this NAF entity instead)
+      this.el.components[this.data.controllerComponent].checkIfControllerPresent = x => x;
+      this.el.components[this.data.controllerComponent].injectTrackedControls = x => x;
+      
+      // we load the model indicated by the remote user's headset
+      this.el.components[this.data.controllerComponent].loadModel({profiles:JSON.parse(this.data.webxrControllerProfiles)});      
+      this.el.play();
```

Comment:
Why do you need to call play yourself here?

Reply:
I call pause on the el before adding the controller component, so that I can manually bypass a part of the component before init() is fired. Then after I've updated the added controller component.

The part I'm bypassing is the part that sets up pose tracking for the controller. The controller components are designed with the assumption that they will only be used locally. I have surveyed the controllers, and found that there are basically two ways to get around this:

1) some controllers are better designed, and I can skip the pose detection from every being activated and just load the model separately.
2) some older ones cannot be run without pose detection being activated; for these, I remove pose detection right after it is added. (that's what the setInterval is for.)